### ISSUE TITLE
don't rely on git for dependencies

### ISSUE THREAD

**Issue by justinbmeyer** [stealjs/steal#512]
https://github.com/stealjs/steal/blob/master/package.json#L15

is pointing at:

```
"systemjs": "git://github.com/stealjs/systemjs#0.16.6-bitovi.9",
```

This is causing errors for people who don't have git installed on the command line.  Can we publish our own systemjs?

**Comment by justinbmeyer** [comment]
https://gitter.im/donejs/donejs?at=564786ca8b242470793db5ae

**Comment by matthewp** [comment]
Yeah, we can publish under **steal-systemjs** or something like that, will need to do the same for es6-module-loader

**Comment by daffl** [comment]
Here is a list the packages and their releases:

- [es6-module-loader](https://github.com/stealjs/es6-module-loader) has been published as [steal-es6-module-loader](https://www.npmjs.com/package/steal-es6-module-loader/) `v0.16.3-bitovi.5` in the [steal-release branch](https://github.com/stealjs/es6-module-loader/tree/steal-release)

**Comment by daffl** [comment]
This should be fixed via #513
